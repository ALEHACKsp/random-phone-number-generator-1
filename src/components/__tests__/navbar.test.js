import React from 'react';
import enzyme from 'enzyme';
import Navbar from '../navbar';

describe('Home Component', () => {
  it('should render properly', () => {
    const node = enzyme.shallow(<Navbar />);
    expect(node).toMatchSnapshot();
  });
});
