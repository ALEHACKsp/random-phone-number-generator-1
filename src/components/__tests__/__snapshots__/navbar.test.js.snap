// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component should render properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Navbar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <a
          className="navbar-brand"
          href="#!"
        >
          Random Phone Number Generator
        </a>,
        <button
          aria-controls="navbarSupportedContent"
          aria-expanded="false"
          aria-label="Toggle navigation"
          className="navbar-toggler"
          data-target="#navbarSupportedContent"
          data-toggle="collapse"
          type="button"
        >
          <span
            className="navbar-toggler-icon"
          />
        </button>,
      ],
      "className": "navbar navbar-expand-lg my-nav-bar",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Random Phone Number Generator",
          "className": "navbar-brand",
          "href": "#!",
        },
        "ref": null,
        "rendered": "Random Phone Number Generator",
        "type": "a",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "aria-controls": "navbarSupportedContent",
          "aria-expanded": "false",
          "aria-label": "Toggle navigation",
          "children": <span
            className="navbar-toggler-icon"
          />,
          "className": "navbar-toggler",
          "data-target": "#navbarSupportedContent",
          "data-toggle": "collapse",
          "type": "button",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "navbar-toggler-icon",
          },
          "ref": null,
          "rendered": null,
          "type": "span",
        },
        "type": "button",
      },
    ],
    "type": "nav",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <a
            className="navbar-brand"
            href="#!"
          >
            Random Phone Number Generator
          </a>,
          <button
            aria-controls="navbarSupportedContent"
            aria-expanded="false"
            aria-label="Toggle navigation"
            className="navbar-toggler"
            data-target="#navbarSupportedContent"
            data-toggle="collapse"
            type="button"
          >
            <span
              className="navbar-toggler-icon"
            />
          </button>,
        ],
        "className": "navbar navbar-expand-lg my-nav-bar",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Random Phone Number Generator",
            "className": "navbar-brand",
            "href": "#!",
          },
          "ref": null,
          "rendered": "Random Phone Number Generator",
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "aria-controls": "navbarSupportedContent",
            "aria-expanded": "false",
            "aria-label": "Toggle navigation",
            "children": <span
              className="navbar-toggler-icon"
            />,
            "className": "navbar-toggler",
            "data-target": "#navbarSupportedContent",
            "data-toggle": "collapse",
            "type": "button",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "navbar-toggler-icon",
            },
            "ref": null,
            "rendered": null,
            "type": "span",
          },
          "type": "button",
        },
      ],
      "type": "nav",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
